man/cc-test-reporter-format-coverage.1.md

```diff
@@ -21,6 +21,12 @@ Locate, parse, and re-format supported coverage sources.
 Output to *PATH*. If *-* is given, content will be written to *stdout*. Defaults
 to *coverage/codeclimate.json*.
 
+## -p, --prefix *PATH*
+
+The prefix *PATH* to remove from absolute paths in coverage payloads, to make
```

Comment:
I would just say "The prefix to remove..". No one said you have to use the options argument in its description :), and I think it reads a little awkwardly when you do here.